src/PerfView/GcStats.cs

```diff
@@ -32,7 +32,9 @@ public static void ToHtml(TextWriter writer, TraceProcess stats, TraceLoadedDotN
                 writer.WriteLine("<LI><Font size=3 color=\"red\">Warning: Did not recognize the V4.0 GC Information events.  Falling back to V2.0 behavior.</font></LI>");
             if (!string.IsNullOrEmpty(stats.CommandLine))
                 writer.WriteLine("<LI>CommandLine: {0}</LI>", stats.CommandLine);
-            writer.WriteLine("<LI>Runtime Version: {0}</LI>", runtime.RuntimeVersion != null ? runtime.RuntimeVersion : "<Unknown Runtime Version>");
+            var runtimeBuiltTime = "";
+            if (runtime.RuntimeBuiltTime != default(DateTime)) runtimeBuiltTime = string.Format(" (built on {0})", runtime.RuntimeBuiltTime);
+            writer.WriteLine("<LI>Runtime Version: {0}{1}</LI>", runtime.RuntimeVersion != null ? runtime.RuntimeVersion : "<Unknown Runtime Version>", runtimeBuiltTime);
```

Comment:
It makes more sense now, I did not even notice that portion ;)  It frankly is dead code, but as you suggested I am on that line so I will make your suggested update.  Testing now.